streams/src/test/java/org/apache/kafka/streams/integration/KStreamRepartitionIntegrationTest.java

```diff
@@ -90,6 +95,22 @@
     private Properties streamsConfiguration;
     private List<KafkaStreams> kafkaStreamsInstances;
 
+    @Parameters(name = "Optimization = {0}")
+    public static Collection<Object[]> data() {
+        final List<Object[]> values = new ArrayList<>();
+
+        Arrays.asList(StreamsConfig.OPTIMIZE, StreamsConfig.NO_OPTIMIZATION)
+              .forEach(x -> values.add(new Object[]{x}));
+
+        return values;
```

Comment:
Seems unnesseary complex? A simple 
```
        return Arrays.asList(new String[][] {
            {StreamsConfig.OPTIMIZE},
            {StreamsConfig.NO_OPTIMIZATION}
        });
```
would do, too :)

(Feel free to ignore the comment.)